pkg/restore/restore.go

```diff
@@ -1213,123 +1218,71 @@ func (ctx *restoreContext) restoreItem(obj *unstructured.Unstructured, groupReso
 	}
 
 	if groupResource == kuberesource.PersistentVolumes {
-		switch {
-		case hasSnapshot(name, ctx.volumeSnapshots):
-			oldName := obj.GetName()
-			shouldRenamePV, err := shouldRenamePV(ctx, obj, resourceClient)
-			if err != nil {
-				errs.Add(namespace, err)
-				return warnings, errs, itemExists
+		pvCondition := 0
+		if volumeInfo, ok := ctx.volumeInfoMap[obj.GetName()]; ok {
+			ctx.log.Infof("Find VolumeInfo for PV %s.", obj.GetName())
+
+			switch volumeInfo.BackupMethod {
+			case volume.NativeSnapshot:
+				pvCondition = pvHasNativeSnapshot
+			case volume.PodVolumeBackup:
+				pvCondition = pvHasPodVolumeBackup
+			case volume.CSISnapshot:
+				if volumeInfo.SnapshotDataMoved {
+					pvCondition = pvHasDataUpload
+				} else {
+					pvCondition = pvHasCSIVolumeSnapshot
+				}
+			default:
```

Comment:
The default case in the volumeInfo.BackupMethod switch seems useless as the above 3 cases already cover all potential backup method values. 

Seems that Removing the default will clean up the code without changing functionality.

Reply:
In some cases, it's possible that the VolumeInfo's backup method is not set.
For example, the PV data is skipped from the backup.

Reply:
Got it. Could we add the comment on volumeInfo.BackupMethod field to indicate its valid value includes empty if PV data is skipped?

Reply:
Sure.